addons/account/wizard/account_move_send.py

```diff
@@ -231,26 +231,27 @@ def _compute_send_mail_extra_fields(self):
         for wizard in self:
             wizard.enable_send_mail = wizard.mode in ('invoice_single', 'invoice_multi')
             wizard.display_mail_composer = wizard.mode == 'invoice_single'
-            send_mail_readonly = False
 
+    @api.depends('move_ids', 'checkbox_send_mail')
```

Comment:
```suggestion
    @api.depends('mode', 'checkbox_send_mail')
```
By transitivity, you depend of move_ids that way since mode depends of move_ids